frontend/src/reducer/notificationPage.js

```diff
@@ -24,6 +31,11 @@ const defaultNotificationsObject = {
   unreadNotificationCount: 0
 }
 
+const NUMBER_OF_CRITERIA = {
+  ONE: 'oneCriteriaGroup',
+  TWO: 'twoCriteriaGroup'
+}
+
```

Comment:
Since the keys are not particularly informative, I'd just use constants 1 and 2 directly and I would not use such named constants here . An object for this since a bit complicated for no obvious (to me) advantages.

You can keep it if you have a different opinion in the matter though.

Reply:
will fix for the strings, but I will maintain the object